### ISSUE TITLE
Choose a single word-separation style?

### ISSUE THREAD

**Issue by callumlocke** [ft-interactive/g-ukpoliticalparties#1]
The property names have three different word-separation styles (camelCase, underscores, and everythinglowercase) depending on which field you want. Can we just stick to one? Preferably camelCase?

**Comment by tomgp** [comment]
yeah, i used the style from the source data in each case. will change this
